### PR TITLE
Add finish.html that relates to #14

### PR DESCRIPTION
**Pull request by RhodesPeter**
Add the finish page which is issue number #14.

### CODE REVIEW

**Comment by des-des** [comment]
@RhodesPeter pls assign me if you want me to merge

File: finished.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+</head>
+<body>
+  <div class="finish-background">
+    <div class="finish">
+      <div class="finish__text">
+        <h1 class="finished__header">You are finished!</h1>
+        <img class="finish__lion"src="assets/lion-no-tail.svg"></img>
+        <h5 class="finished__header">Download your answers</h5>
+        <a href="#" download="my-questionnaire" target="blank">
+          <input class="download-button" type="button" value="&#8595">
```
Comment (des-des) [inline finished.html]: what is `&#8595`?

**Comment by RhodesPeter** [comment]
Oops I made you a reviewer instead, I've assigned you now.

File: main.css [hunk 2]
```diff
@@ -84,3 +85,51 @@ body {
...
+.download-button:active {
+  box-shadow: 0 2px #E87325;
+  transform: translateY(2px);
+}
+.finish__lion {
+  width: 300px;
+  display: block;
+  margin: 40px auto;
+}
+h1 {
```
Comment (des-des) [inline main.css]: You could consider giving these classes?

File: main.css [hunk 3]
```diff
@@ -84,3 +85,51 @@ body {
...
+  margin-top: 0;
+  margin-bottom: 15px;
+}
+h5 {
```
Comment (des-des) [inline main.css]: and this one

File: finished.html (hunk 1 above)
Comment (RhodesPeter) [inline finished.html]: This is an arrow. I found it here --> http://htmlarrows.com/

File: main.css (hunk 3 above)
Comment (RhodesPeter) [inline main.css]: Ok no problem. I will give this a go now 👍

**Comment by RhodesPeter** [comment]
I have made these alterations @des-des.

File: finished.html (hunk 1 above)
Comment (des-des) [inline finished.html]: ok cool, fyi its good to leave a comment for stuff like that

**Pull request merged**